assets/css/global.less

```diff
@@ -1970,6 +1970,13 @@ body .guides .guides-section-white {
   }
   h1, h2, h3, h4, h5, h6 {
     color: @guides-font;
+    margin: 10px 0; 
+    &:hover > a {
```

Comment:
Where does the `a` come from? Is that added automatically by JS?

Reply:
Jekyll adds an anchor tag to all headers